keyboards/bandominedoni/keymaps/via/keymap.c

```diff
@@ -28,6 +29,10 @@ enum layer_names {
     _FN
 };
 
+enum custom_keycodes {
+    VERSION = SAFE_RANGE
```

Comment:
This would allow you to list it in VIA
```suggestion
    VERSION = USER00
```

Reply:
Wow! That’s cool. We’ve been waiting for such a feature on VIA!
I’ll update my codes as you suggested. 

It seems you can use this as long as you have 16 or less custom codes. 
https://github.com/qmk/qmk_firmware/blob/master/quantum/via.h